front/app/components/UI/SearchInput/index.tsx

```diff
@@ -41,9 +40,6 @@ const SearchInputWrapper = ({
   labelColor,
 }: Props & WrappedComponentProps) => (
   <>
-    <Label htmlFor="search-input" hidden>
-      {formatMessage(messages.searchLabel)}
-    </Label>
     <SearchInput
```

Comment:
This label wasn't necessary and is already covered by the  `ariaLabel={ariaLabel || formatMessage(messages.searchAriaLabel)}` (line 47).